docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,129 @@
 
-In this recipe we will learn how to configure and use S3cmd to manage data with Minio Server.
+This guide describes how to configure `S3cmd` and use it to manage data on Minio Server. These are the steps you will follow:
 
-## 1. Prerequisites
+[1. Install Minio Server](#installminioserver)<br/>
+[2. Install `S3cmd`](#installs3cmd)<br/>
+[3. Configure `S3cmd`](#configures3cmd)<br/>
+[4. Run `S3cmd` Commands](#runs3cmdcommands)
 
-Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
+## <a name="installminioserver"></a>1. Install Minio Server
 
-## 2. Installation
+Install Minio Server using the instructions in the [Minio Quickstart Guide](http://docs.minio.io/docs/minio-quickstart-guide).
 
-Install `S3cmd` from <http://s3tools.org/s3cmd>.
+## <a name="installs3cmd"></a>2. Install `S3cmd`
 
-## 3. Configuration
+Install `S3cmd` using these instructions: <http://s3tools.org/s3cmd>.
 
-We will run `S3cmd` on <https://play.minio.io:9000>.
+## <a name="configures3cmd"></a>3. Configure `S3cmd`
 
-Access credentials shown in this example belong to <https://play.minio.io:9000>. These credentials are open to public. Feel free to use this service for testing and development. Replace with your own Minio keys in deployment.
+### 3.1 Generate a Configuration file
+`S3cmd` uses a configuration file called **.s3cfg** to access cloud storage. Use the following command to generate the initial version of **.s3cfg**:
 
-Edit the following fields in your s3cmd configuration file `~/.s3cfg`
+```sh
+ `./S3cmd --configure`
+```
+
+**Note:** accept the defaults when prompted.
+
+### 3.2 Edit the Configuration File
+Modify the configuration file to enable `S3cmd` to manage buckets on https://play.minio.io:9000:
+
+#### 3.2.1. Navigate to **/users/<your user name>** and open **.s3cfg** in a text editor.
+#### 3.2.2. Edit the following fields in **.s3cfg** to configure the endpoint:
 
 ```sh
-# Setup endpoint
+bucket_location = us-east-1
 host_base = play.minio.io:9000
 host_bucket = play.minio.io:9000
-bucket_location = us-east-1
 use_https = True
+```
 
-# Setup access keys
+#### 3.3.3. Edit the following fields in **.s3cfg** to set the access keys:
+
+```sh
 access_key =  Q3AM3UQ867SPQQA43P2F
 secret_key = zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
+```
+
+#### 3.3.4. Edit the following fields in **.s3cfg** to enable S3 v4 signature APIs:
 
-# Enable S3 v4 signature APIs
+```sh
 signature_v2 = False
 ```
 
-## 4. Commands
+#### 3.3.5. Save **.s3cfg**.
+
+**Note:** The variables set in this example are for public testing and development on <https://play.minio.io:9000>. Modify these variables as appropriate when developing for your own Minio Server.
+
+
+## <a name="runs3cmdcommands"></a>4. Run `S3cmd` Commands
+Navigate to the installation directory for **S3cmd** and invoke the following commands to create and manage a bucket:
 
-### To make a bucket
+### Make a Bucket
 
 ```sh
 s3cmd mb s3://mybucket
```

Comment:
All other examples use s3://testbucket, should change this to match

Reply:
same comment as @eco-minio

Reply:
@deekoder , @eco-minio  I have made these changes in the branch. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-2/docs/s3cmd-with-minio.md.